lib/linux.js

```diff
@@ -32,12 +35,18 @@ const parseLinux = ({stdout, boundsStdout, activeWindowId}) => {
 	const windowId = (resultKeys.indexOf(windowIdProperty) > 0 &&
 		parseInt(result[windowIdProperty].split('#').pop(), 16)) || activeWindowId;
 
+	const processId = parseInt(result['_NET_WM_PID(CARDINAL)'], 10);
+
+	if (isNaN(processId)) {
+		throw new Error('Failed to parse process id'); // eslint-disable-line unicorn/prefer-type-error
```

Comment:
```suggestion
		throw new Error('Failed to parse process ID'); // eslint-disable-line unicorn/prefer-type-error
```

Reply:
Why is this suddenly throwing?

Reply:
Errors are caught later in any case, and if we do
```js
return undefined; // Failed to parse process ID
```
here, we will run into weird errors when augmenting the result later or will have to check for `undefined` when accessing fields later.
IMO it's much clearer this way.